pandas/core/style.py

```diff
@@ -255,6 +272,75 @@ def _translate(self):
                     precision=precision, table_styles=table_styles,
                     caption=caption, table_attributes=self.table_attributes)
 
+    def format(self, formatter, subset=None):
+        '''
+        Format the text display value of cells.
+
+        .. versionadded:: 0.18.0
+
```

Comment:
Ah, I was talking about a method on `DataFrame`. I could see something like what you're talking about being useful. I'll play around with it.